CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dev
+
+## Fix 
+
+- Hide disabled operations from Instance view
```

Comment:
and the line that correspond to this feature?

Reply:
done